pylibs/setup.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020, The OTNS Authors.
```

Comment:
nit: add an empty line before the head of copyright, like the OpenThread example [here](https://github.com/openthread/openthread/blob/master/tests/scripts/thread-cert/Cert_5_1_07_MaxChildCount.py).

Reply:
Sure, but it's better to do it in a separate PR?